snuba/web/db_query.py

```diff
@@ -408,10 +408,10 @@ def execute_query_with_caching(
     )
 
     with sentry_sdk.start_span(description="execute", op="db") as span:
+        key = get_query_cache_key(formatted_query)
```

Comment:
There was a test failing that uses this code, and I had to make this change to ensure query_id is populated.

I don't think we actually use this function any more except in tests though.